plugins/heroku/heroku_log_input.go

```diff
@@ -0,0 +1,204 @@
+/***** BEGIN LICENSE BLOCK *****
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# The Initial Developer of the Original Code is the Mozilla Foundation.
+# Portions created by the Initial Developer are Copyright (C) 2014
+# the Initial Developer. All Rights Reserved.
+#
+# This plugin is based on the HttpListenPlugin.
+#
+# Contributor(s):
+#   Anton Lindström (carlantonlindstrom@gmail.com)
+#
+# ***** END LICENSE BLOCK *****/
+
+package heroku
+
+import (
+	"code.google.com/p/go-uuid/uuid"
+	"fmt"
+	"github.com/mozilla-services/heka/message"
+	"github.com/mozilla-services/heka/pipeline"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type HerokuLogInput struct {
+	conf        *HerokuLogInputConfig
+	listener    net.Listener
+	stopChan    chan bool
+	ir          pipeline.InputRunner
+	server      *http.Server
+	starterFunc func(hli *HerokuLogInput) error
+	hostname    string
+}
+
+const LOGPLEX_CONTENT_TYPE = "application/logplex-1"
+
+// Heroku Log Input config struct
+type HerokuLogInputConfig struct {
+	// TCP Address to listen to
+	// Defaults to "0.0.0.0:8325".
+	Address string
```

Comment:
Did you tried to parametrize this module via config? Looks like there's missing `toml:` tag